### PR TITLE
feat(ci): add GH actions release pipeline

### LINKED ISSUE #7: Move nri-kube-events-release and nr-kube-events-tests Jobs
Old Jenkins job
https://fsi-build.pdx.vm.datanerd.us/view/all/job/nri-kube-events-release/

https://fsi-build.pdx.vm.datanerd.us/view/all/job/nr-kube-events-tests/

 Notice that: 
 - The tests is described in the Jenkins file stored in the repo 
 - The release in the job option directly in jenkins

We should make sure we follow a common pattern:
PR/Merge -> Run tests/build using a makefile
Create Tag -> Push release

NOTE: remove old jobs after this is merged

### PR DESCRIPTION
**Pull request by arvdias**
Adds GH actions release workflow:
 - builds docker image and published it to docker hub
 - copies k8s manifest to S3
 - supports building pre-release and changing from pre-release to release

Fixes #7

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v1 
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_PASSWORD }}
+
+      - name: Build and push
+        run: |
+          make docker-build
```
Comment (roobre) [inline .github/workflows/release.yml:48]: We might be missing to push `:latest` image here. On k8s-metadata-injection was missing and I added it this very morning: https://github.com/newrelic/k8s-metadata-injection/pull/73/commits/e393e1d3a36a1faae5bc7dd5010e629f0a61e087

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    types:
+      # works for building pre-releases
+      - published
+      # works for releases and changing from pre-release to release
+      - released
+
+env: 
+ 
+  DOCKER_IMAGE_NAME: newrelic/nri-kube-events
+  RELEASE_VERSION: ${{ github.event.release.name }}
```
Comment (roobre) [inline .github/workflows/release.yml:13]: I have a slight preference for `${{ github.event.release.tag_name }}`, as the release name is a bit more arbitrary (and visible only in GH UI, as opposed to the tag which is built into the git tree)

**Comment by arvdias** [comment]
closing in favor of https://github.com/newrelic/nri-kube-events/pull/10

**Comment by arvdias** [comment]
Closed this PR as i removed the repository by mistake :(

**Pull request closed without merging**